kratos/processes/levelset_forward_convection_process.h

```diff
@@ -0,0 +1,581 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Simon Wenczowski
+//
+
+#if !defined(KRATOS_LEVELSET_FORWARD_CONVECTION_PROCESS_INCLUDED )
+#define  KRATOS_LEVELSET_FORWARD_CONVECTION_PROCESS_INCLUDED
+
+// System includes
+#include <string>
+#include <iostream>
+#include <algorithm>
+
+// External includes
+
+// Project includes
+#include "includes/convection_diffusion_settings.h"
+#include "includes/define.h"
+#include "includes/kratos_flags.h"
+#include "elements/levelset_convection_element_simplex.h"
+#include "geometries/geometry_data.h"
+#include "solving_strategies/schemes/residualbased_incrementalupdate_static_scheme.h"
+#include "solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h"
+#include "solving_strategies/strategies/residualbased_linear_strategy.h"
+#include "utilities/variable_utils.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/// Short class definition.
+/** This function operates on a given distance field and given velocity field at time t.
+ *  Based on the velocity, the distance field is convected forward and reaches its state at time t + dt.
+ *  The artificial time step dt can be chosen freely and does not coincide with the time step of the simulated physical time.
+ *  The convected distance field also does NOT overwrite the actual distace field but is stored in a seperate variable.
+ *  Thus, the artificially convected distance filed can be used e.g. for the local mass conservation correction.
+ *
+*/
+template< unsigned int TDim, class TSparseSpace, class TDenseSpace, class TLinearSolver >
+class LevelSetForwardConvectionProcess : public Process
```

Comment:
this has no check in the Core

I see that there is one in Fluid, but cannot this be used standalone?